torch/_numpy/random.py

```diff
@@ -82,7 +100,7 @@ def uniform(low=0.0, high=1.0, size=None):
         size = ()
     dtype = _dtypes_impl.default_dtypes().float_dtype
     values = torch.empty(size, dtype=dtype).uniform_(low, high)
-    return array_or_scalar(values, return_scalar=size is None)
+    return array_or_scalar(values, return_scalar=size == ())
 
 
 @deco_stream
```

Comment:
In this one we should probably just pass `return_scalar=False`.

Reply:
Also why? In numpy `uniform(0, 1) ` does returns python float

Reply:
I commented here referring to the function below, as I couldnt' comment in the lines below